Source/WebKit/UIProcess/Network/NetworkProcessProxyCocoa.mm

```diff
@@ -58,7 +58,7 @@
     if (!messageName || !*messageName)
         return false;
 
-    if (LaunchServicesDatabaseXPCConstants::xpcLaunchServicesDatabaseXPCEndpointMessageName == messageName) {
+    if (!strcmp(messageName, NetworkProcessEndpointXPCConstants::xpcNetworkProcessXPCEndpointMessageName)) {
```

Comment:
The previous code was more readable. Why did you make this change? ASCIILiteral has an operator==() specifically for this.